projects/js-packages/components/components/admin-section/hero/index.jsx

```diff
@@ -17,12 +20,14 @@ import './style.scss';
 const AdminSectionHero = props => {
 	const { children } = props;
 	return (
-		<div className="jp-admin-section-hero">
-			<div className="jp-wrap">
-				<div class="jp-row">
-					<div class="lg-col-span-12 md-col-span-8 sm-col-span-4">{ children }</div>
-				</div>
-			</div>
+		<div className={ styles[ 'jp-admin-section-hero' ] }>
+			<Wrap>
```

Comment:
Looking now into this structure, looks like `Wrap` (or `Container`) could accept an `className` and will have one less component into our tree.

```javascript
<Container className={ styles[ 'jp-admin-section-hero' ] }>
  <Row>...</Row>
</Container>
```

Reply:
but do we want to expose className to the world?

Reply:
Don't think that's a problem, we just apply the component classes after to make ours with more priority.

Ex: https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/button/index.js#L95-L106

Reply:
Actually just realized we need the additional element there.

That's because we need that div to apply the background to 100% of the width of the screen, while `Wrap` (or `Container`) limits the width to the content area.

Here's an example of how it looks like if I did that to AdminSection (but not to AdminSectionHero):

![Captura de Tela 2021-12-08 às 12 07 44](https://user-images.githubusercontent.com/971483/145232056-b9727b55-8631-4497-8b4b-8ab056e1d39c.png)

Let me know if you think there's a better solution for that

Reply:
@leogermani gotcha! let's move this way, don't think we have some different approach for that.